src/schema/resolvers.js

```diff
@@ -1,8 +1,38 @@
+const _ = require('lodash');
 const pubsub = require('../pubsub');
 const fetch = require('node-fetch');
 
-function buildTopicFilters({OR = [], address, status}) {
-  const filter = (address || status) ? {} : null;
+/* Default limit number for query page */
+const DEFAULT_LIMIT_NUM = 50;
+const DEFAULT_SKIP_NUM = 0;
+
+/**
+ * Returns an options object to pass in database query
+ * @param  {Array} orderBy Array of Order type defined in schema
+ * @param  {Number} limit   limit number in paging; default value DEFAULT_LIMIT_NUM
+ * @param  {Number} skip    skip number in paging; default value DEFAULT_SKIP_NUM
+ * @return {Object}         
+ */
+function buildQueryOptions(orderBy, limit, skip) {
+  const options = {};
+
+  if (!_.isEmpty(orderBy)) {
+    options.sort = [];
+
+    _.each(orderBy, (order) => {
+      options.sort.push([order.field, (order.direction === "ASC" ? 'asc' : 'desc')]);
```

Comment:
according to https://github.com/louischatriot/nedb sort section, nedb supports `{field: 1/-1}`, not ASC, DESC

Reply:
👍 removed